go/private/rules/cgo.bzl

```diff
@@ -523,6 +523,8 @@ def setup_cgo_library(name, srcs, cdeps, copts, cxxopts, cppopts, clinkopts, obj
 
   if objc:
     cgo_objc_lib_name = name + ".cgo_objc_lib"
+    kwargs = extra_args
+    kwargs.update(objcopts)
```

Comment:
I think you need to update `objcopts` with `extra_args` here instead of updating `extra_args`. This statement modifies `extra_args`, which is used in other rules below.

Reply:
Oh, right! Thanks for catching this.